common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/Image.scala

```diff
@@ -30,8 +30,39 @@ case class MergedImage[Id <: IdState](
       id = id,
       location = location
     )
+
+  def augment(inferredData: => InferredData): AugmentedImage[Id] =
+    augmentWithOption(Some(inferredData))
+
+  def augmentWithNone: AugmentedImage[Id] = augmentWithOption(None)
```

Comment:
Seems like not a massively convenient convenience method?

Reply:
It is useful (when we fail to infer data) but do you have a suggestion of a better name?

Reply:
Not sure why just having `augment(data: Option[Data])` and `augment(None)` isn't as clear as what you have here?